### ISSUE TITLE
Combat quest: crash quickly

### ISSUE THREAD

**Issue by guenael44** [CitizensDev/Citizens#319]
Hello i have special demand 

I have 3 groupe : Groupe1 Groupe2 Groupe3
I want when groupe3 kill Groupe1 or Groupe2, they earn 1 kill on combat quest

i have do that :

[CODE]pvp1kill:
    texts:
        description: Quete PvP- /Tuer un allié/ Vous voici sur la terre de nos ennemis, trouvez en un et revennez me voir.(Tuez un membre de l'alliance)
        completion: <g>Bravo, un de moins, <player>. Vous recevez 20 Adelias et 20 flêches.
        acceptance: <g>Quete <y>acceptée<g>.
    repeats: 0
    requirements:
        '0':
            type: rank
            rank: Groupe3
    objectives:
        '0':
            '0':
                type: player combat
                amount: 1
                string: 'g:Groupe1, g:Groupe2'
                message: Allié mort; 1/1. Retournez voir Goblin.
    rewards:
        '0':
            type: money
            money: 20
        '1':
            type: item
            id: 262
            amount: 20
            take: false
[/CODE]

but when i kill somebody nothing happen ?

Others quest type working
Groupe1 Groupe2 Groupe3 : existe in bpermission and seem string: '*' don't work too

-------------- EDIT -----

Ok i find some evidences :

All work well but if i kill more than 2 player , citizens ignore all others kill (count 2 more but display nothing), except if i restart the server, it continue to count AND display my kill and the 2 last kill before i restart . And the quest is validate after reboot (again) on final kill

**Comment by fullwall** [comment]
Check your quest combat exploit settings.

**Comment by guenael44** [comment]
can you help me more, i don't understand what is going wrong

**Comment by fullwall** [comment]
Can you post your config.yml?

**Comment by guenael44** [comment]
range:
    look: 5
    pathfinding: 16.0
items:
    select-items: '*'
    talk-items: 340,
ticks:
    saving:
        save-often: true
        use-task: true
        delay: 72000
    pathing:
        max-stationary: -1
        max-pathing: -1
    waypoints:
        right-click-pause: 70
economy:
    prices:
        basic:
            creation: 100
        blacksmith:
            creation: 100
            toolrepair:
                wood: 0.15
                diamond: 0.75
                stone: 0.45
                gold: 0.3
                iron: 0.6
                misc: 1.0
            armorrepair:
                diamond: 0.75
                gold: 0.3
                leather: 0.15
                iron: 0.6
                chainmail: 2.25
        guard:
            creation: 100
        healer:
            creation: 100
            heal: 100
            levelup: 100
        quester:
            creation: 100
        trader:
            creation: 100
        wizard:
            creation: 100
            spawnmob: 100
            changetime: 100
            togglestorm: 100
            teleport: 100
    use-economy: true
debug:
    enable-error-reporting: true
    debug-mode: false
    error-reporting-ident: Anonymous
general:
    defaults:
        enable-following: true
        talk-when-close: false
    quick-select: false
    colors:
        use-npc-colours: true
        npc-colour: f
    notify-updates: true
    chat:
        space-char: /
        selection-message: <g>Vous avez selectionné <y><npc><g> (ID <y><npcid><g>).
        default-text: Hello.;How are you today?;Having a nice day?;Good weather today.;Stop hitting me!;I'm bored.;
        creation-message: <g>The NPC <y><npc><g> was born!
        format: '[%name%]: '
    pvp:
        deny:
            attack-disabled-pvp-players: true
guards:
    bouncers:
        default:
            protection-radius: 10
    max:
        stationary-return-ticks: 25
    respawn-delay: 100
healers:
    give-health-item: 35
    take-health-item: 276
    health-regen-increment: 12000
    regen-health: true
wizards:
    mana-regen-item: 348
    regen-mana: true
    max-mana: 100
    max-locations: 10
    mana-regen-rate: 6000
    interact-item: 288
    mana-costs:
        spawnmob: 5
        changetime: 5
        togglestorm: 5
        teleport: 5
quests:
    exploits:
        item-pickup:
            check-delay: 1200
        combat:
            check-times: 2
            check-radius: 20
        blocks:
            tracking-remove-delay: 6000
    save:
        command-delay-ms: 5000

**Comment by guenael44** [comment]
i think i understand , it's combat:
check-times: 2
check-radius: 20 ?

Can you explain why just "2" and not unlimited check time ? Cpu or memory problem could happen if more ? Same question for radius, thx :)

**Comment by fullwall** [comment]
No, there's little CPU overhead. Those are just some defaults I picked randomly.
